thirdparty/assimp/assimp.qbs

```diff
@@ -59,7 +59,7 @@ Project {
         Depends { name: "pugixml" }
 
         cpp.includePaths: assimp.incPaths
-        cpp.cxxLanguageVersion: assimp.languageVersion
+        cpp.cxxLanguageVersion: "c++17"
```

Comment:
languageVersion defined in Thunder.qbs please use it

Reply:
Grabbing coffee. Fixing now.

Reply:
Are you asking me to use c++14 or change it globally to c++17? If it needs to be c++14, then there will be other changes that need to be made instead.

Reply:
Threw in a typo fix with this that I saw while I was out. Last night's CMake fix tagged along with this one. Hopefully, it doesn't cause a merge issue. I'm not sure how to not have that happen sometimes.

Reply:
Global c++17 is good for me. but seems like not for Visual Studio let me check this